### DISCUSSION TITLE
Backup de Nuvem (Open Source?)

### DISCUSSION THREAD

**Discussion by AdrielGraminho** [frontendbr/forum#2454] [Discussão]
Alguém conhece alguma ferramenta open source para gerenciar backup na nuvem?
Eu conheço algumas como o Veeam mas nada Open Source...
Isso faz eu como dev ver uma lacuna no mercado e já querer criar uma poc disso...

**Comment by igorlvicente** [comment]
NextCloud? SyncThing? Depende do que exatamente você quer fazer.

**Reply by AdrielGraminho** [reply]
quero fazer backup de uma infraestrutura na AWS com diversos discos, isso inclui RDSs, EC2...
